mod_reforged/hooks/skills/actives/charge.nut

```diff
@@ -0,0 +1,47 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/charge", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
+		this.m.Description = "Hurl yourself forward, crashing into your enemies with overwhelming force!";
+	}
+
+	// Vanilla has a getTooltip function defined for this skill but it doesn't provide all the details
+	// so we overwrite it to produce a better tooltip overall
+	q.getTooltip = @() function()
+	{
+		local ret = this.getDefaultUtilityTooltip();
+		ret.extend([
+			{
+				id = 10,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Move to the target tile")
+			},
+			{
+				id = 11,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Has a " + ::MSU.Text.colorPositive("100%") + " chance to stun adjacent enemies upon arrival, reduced by the defense granted by target\'s shield and [Shieldwall|Skill+shieldwall_effect]")
```

Comment:
```suggestion
				text = ::Reforged.Mod.Tooltips.parseString("Has a " + ::MSU.Text.colorPositive("100%") + " chance to stun a random adjacent enemy upon arrival, reduced by the defense granted by target\'s shield and [Shieldwall|Skill+shieldwall_effect]")
```
This effect only chooses a single valid victim for potentially receiving the stun